packages/styled-components/src/utils/stylis.js

```diff
@@ -11,21 +11,8 @@ export type Stringifier = (
   componentId: string
 ) => Array<string>;
 
-export default function createStylisInstance(
-  options: Object = EMPTY_OBJECT,
-  plugins: Array<Function> = EMPTY_ARRAY
-) {
-  const stylis = new Stylis({
-    global: false,
-    cascade: true,
-    keyframe: false,
-    prefix: true,
-    compress: false,
-    semicolon: false, // NOTE: This means "autocomplete missing semicolons"
```

Comment:
the only change from this current set of options is `semicolon` is turned on

Reply:
I think that's technically a breaking change that might affect plenty of apps without reasonable errors/warnings in place 😅

Reply:
I've counted with this problem when updating from 4.x to 5.x just because of this config change.  'Not autocomplete missing semicolons' makes some styles can't be compiled right as before. Though I can add semicolons manually when I'm writing new code, but I'm not sure how many old codes are effected by this change. Can we change the default value of `semicolon` to `true`? Though It might cause a little growing in bundle size, but we can get a more smoothing migration experience from 4.x to 5.x.